expo/gbe/views/edit_act_techinfo_view.py

```diff
@@ -31,10 +31,13 @@
     LightingInfoForm,
     CueInfoForm,
     VendorCueInfoForm,
-
-)
+    )
 from scheduler.models import Event as sEvent
 from gbetext import default_update_act_tech
+from expo.settings import (
+        DATETIME_FORMAT,
+        DAY_FORMAT,)
```

Comment:
It is passing TIME_FORMAT into date_format as a literal string, which imports the standard time formats from settings.py and uses eval to get their value (not how I would do it).  We need to import the formats if we are using non-standard ones or are not passing into date_format as text.